scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -1055,8 +1055,9 @@ def build(self):
         harness = HarnessImporter.get_harness(self.instance.testsuite.harness.capitalize())
```

Comment:
could be fine to keep harness in `self` variables, because it is also created in `run` method (as a new instance)